x-pack/platform/plugins/shared/fleet/server/services/agent_policy.ts

```diff
@@ -913,13 +913,18 @@ class AgentPolicyService {
     soClient: SavedObjectsClientContract,
     esClient: ElasticsearchClient,
     id: string,
-    options?: { user?: AuthenticatedUser; removeProtection?: boolean; asyncDeploy?: boolean }
+    options?: {
+      user?: AuthenticatedUser;
+      removeProtection?: boolean;
+      asyncDeploy?: boolean;
+      skipValidation?: boolean;
+    }
   ): Promise<void> {
     return withSpan('bump_agent_policy_revision', async () => {
       await this._update(soClient, esClient, id, {}, options?.user, {
         bumpRevision: true,
         removeProtection: options?.removeProtection ?? false,
-        skipValidation: false,
+        skipValidation: options?.skipValidation ?? false,
```

Comment:
Is there any scenario where we want to bump revision and perform validation? I am wondering if we should `skipValidation` for all bumpRevision

Reply:
You are right, we can probably skip validation for all bumpRevision calls